plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -85,7 +85,16 @@ static const char *my_dhm_G = "4";
 #include <openssl/bio.h>
 #include <openssl/buffer.h>
 #endif
+#endif
 
+#if defined(TCP_USER_TIMEOUT)
+#define SOCKET_LEVEL IPPROTO_TCP
+#define SOCKET_TIMEOUT_OPT TCP_USER_TIMEOUT
+#define SOCKET_TIMEOUT_VAR(s) int to = s*1000
+#else
+#define SOCKET_LEVEL SOL_SOCKET
+#define SOCKET_TIMEOUT_OPT SO_SNDTIMEO
+#define SOCKET_TIMEOUT_VAR(s) SET_RCVTIMEO(to, s)
 #endif
```

Comment:
Not a fan of the `SOCKET_TIMEOUT_VAR` style of setting the timeout. It makes the
```SOCKET_TIMEOUT_VAR(r->Link.sendTimeout);```
line later on in the code impossible to understand without having to refer back to the define. I understand the desire to avoid code duplication but I think this makes it too unclear what's going on.

Reply:
I've updated the macro here to align with the `SET_RCVTIMEO(tv, s)` macro.